### ISSUE TITLE
Show what will/won't be committed same as default git commit template

### ISSUE THREAD

**Issue by yyyc514** [divmain/GitSavvy#107]
IE:

```
Fix: this commit fixes xyz

## To make a commit, type your commit message and press SUPER-ENTER. To cancel
## the commit, close the window. To sign off the commit press SUPER-S.

## You may also reference or close a GitHub issue with this commit.  To do so,
## type `#` followed by the `tab` key.  You will be shown a list of issues
## related to the current repo.  You may also type `owner/repo#` plus the `tab`
## key to reference an issue in a different GitHub repo.

# On branch master
# Your branch is up-to-date with 'origin/master'.
# 
# Changes to be committed:
# 
# 	new file:   project.sublime-project
# 
# Untracked files:
# 
# 	project.sublime-workspace
```

We can get this from `git commit --dry-run` with a few small tweaks.  I like the single pounds to differentiate it from the above help, but we could also use ##.

**Comment by yyyc514** [comment]
It would be nice to get the meat (what's changed) a little higher... could the notes about github issues be placed below?  Thoughts?

**Comment by yyyc514** [comment]
https://github.com/divmain/GitSavvy/pull/111

**Comment by p3lim** [comment]
Try setting the option `show_commit_diff` to true, it'll show a diff for the commit.

**Comment by yyyc514** [comment]
Perfect IF what I wanted was a diff.  What I'm going for is to match the default git behavior for editing commits.

**Comment by p3lim** [comment]
Ahh, now I know what you mean.

**Comment by divmain** [comment]
I'm going to tackle this myself.  I don't see any reason why it can't be toggle-able, ala the remote branches in the `git: branch` dashboard.  Probably the same with the diff.  I'll include syntax highlighting in that effort.

**Comment by yyyc514** [comment]
I'm open to helping with the larger refactoring also.  Just trying to hit the ground running on a few things and I didn't realize there was a whole new interface paradigm until you pointed it out. :)

**Comment by divmain** [comment]
This will be included as part of #50.  Closing in favor of that issue.
